include/fmt/format.h

```diff
@@ -2627,11 +2627,11 @@ class format_string_checker {
   FMT_CONSTEXPR void on_text(const Char*, const Char*) {}
 
   FMT_CONSTEXPR void on_arg_id() {
-    arg_id_ = context_.next_arg_id();
+    arg_id_ = static_cast<unsigned>(context_.next_arg_id());
```

Comment:
This cast won't be needed with the change mentioned earlier.

Reply:
This is a different arg_id_ ; do you want me to convert it to 'int' as well?

Reply:
Done.